riscv_ctg/data/imc.yaml

```diff
@@ -1862,9 +1867,11 @@ c.jalr:
   isa: 
     - IC
   formattype: 'crformat'
+  rs1_val_data: 'gen_sign_dataset(xlen)+ gen_sp_dataset(xlen)'
+  rs2_val_data: 'gen_sign_dataset(xlen)+ gen_sp_dataset(xlen)'
   template: |-
 
     // $comment
-    // opcode:c.jalr; op1:$rs1
-    TEST_CJALR_OP($testreg, $rs1, $swreg, $offset)
+    // opcode: c.jr; op1:$rs1; op2:$rs2; op1val:$rs1_val; op2val:$rs2_val
```

Comment:
These changes do not look correct. Were the macro files changed? If so, they aren't included in the PR.

Reply:
Reverted the change since the template did not require the change.